features/step_definitions/chart_of_accounts/account.rb

```diff
@@ -520,6 +520,32 @@
   end
 end
 
+And /^I submit the Account document addressing Continutaion Account errors$/ do
+  # Editting an existing account could cause business rule error that continuation account is closed or expired
+  # upon submit. When those business rule errors occur, edit the continuation account on the account to the default
+  # continutation account parameter value.
+  step 'I submit the Account document'
+
+  #getting errors from page is expensive, obtain reference once that can be reused
+  page_errors = $current_page.errors
+  #search errors array for Continuation Account in any of the error messages
+  continutation_acct_error_exists = false
```

Comment:
There's no need to initialize this given the way you set it in the next line. The value will always be overwritten regardless of the outcome from `#any?`